lib/modules/manager/maven/update.spec.ts

```diff
@@ -4,6 +4,7 @@ import * as pomUpdater from './update';
 
 const simpleContent = Fixtures.get(`simple.pom.xml`);
 const minimumContent = Fixtures.get(`minimum.pom.xml`);
+const prereleaseContent = Fixtures.get(`prerelease.pom.xml`);
```

Comment:
can't you use / update one of the existing fixtures instead?

Reply:
Yes we can certainly update an existing fixture however I think it would be good to have something to isolate prerelease versions. It may have some confusing/undesired effects on existing test cases as the major/minor/patch version will remain the same:
```
$ semver -i major 1.0.0-1
1.0.0
$ semver -i minor 1.0.0-1
1.0.0
$ semver -i patch 1.0.0-1
1.0.0
$ semver -i prerelease 1.0.0-1
1.0.0-2
```

I am not that familiar with the test suite so please let me know if I should update a different fixture instead.

Reply:
Good to have a test against a prerelease version

Reply:
i think here's something wrong with the test.

the fixture has version `1.0.0-1` but the test uses `0.0.1-1` shouldn't they match?
i would expect the final version bumped to `1.0.0-2` instead of `0.0.1-2`

Reply:
Yes, you're right! I missed that part